### PR TITLE
[Design adjustments] Replace RibbonIcon with ClipboardIcon

### LINKED ISSUE #1746: [Design adjustments] Replace Ribbon icon with Clipboard icon
Replace the ribbon icon with the clickboard icon from the ui-kit

Bildschirmfoto 2023-07-10 um 10 11 49

### PR DESCRIPTION
**Pull request by timonrey**
closes #1746

### CODE REVIEW

**Review by gabriele-ct — APPROVED** [review]

File: packages/ui-kit/src/components/markdown.tsx [hunk 1]
```diff
@@ -386,7 +386,7 @@ const withCopyToClipboard =
           onClose={handleTooltipClose}
         >
           <CopyArea onClick={handleCopyToClipboardClick}>
-            <RibbonSvgIcon />
+            <ClipboardIcon size="big" color="neutral60" />
```
Comment (timonrey) [inline packages/ui-kit/src/components/markdown.tsx:389]: "neutral60" is the most suited color provided by the ui-kit for their icons...also very close to the color code you provided @zbalek. And I assume all of their color codes for icons are following the WCAG. But maybe you can double check if the color is fine.
https://uikit.commercetools.com/?path=/story/basics-tokens--all-tokens (color-neutral-60)

**Comment by zbalek** [comment]
Hi @timonrey - unfortunately color-neutral-60 [#999999] doesn't pass the checks

#999999 = https://webaim.org/resources/contrastchecker/?fcolor=999999&bcolor=FFFFFF
#949494 = https://webaim.org/resources/contrastchecker/?fcolor=949494&bcolor=FFFFFF

image

**Comment by zbalek** [comment]
Hi @timonrey - what we can also do is to use the following color from the MC Redesign library: color-neutral-40 and adjust the opacity to 70%. (This would essentially get us the #949494 color.)

**Comment by timonrey** [comment]
I added the #949494 color to the design system now and the icon is using this color. 👍🏼

**Pull request merged**